src/cli/main.py

```diff
@@ -0,0 +1,79 @@
+"""
+Usage: instantshare.py <command> [<args>...]
+       instantshare.py [-h | --help | -v | --version]
+       instantshare.py help <command>
```

Comment:
As you already mentioned in #18, we want to create an executable file for Windows in the long run, so you should change the name of the program to "instantshare" instead of "instantshare.py" and rename the file "instantshare.py" to "instantshare". This prevents inconsistency with the usage documentation later on when using different platforms.

The shebang `#!/usr/bin/env python3` allows executing the script with the correct interpreter on linux by just executing it and for now it is also possible to execute the script on windows by using "python instantshare" (even without the .py file extension).